app.js

```diff
@@ -83,17 +87,25 @@ const sessionOpts = Object.assign({
 
 app.use(session(sessionOpts));
 
+// check for cookies
+app.use(require('hof').middleware.cookies());
+
 // apps
 app.use(require('./apps/rtm/'));
 
 app.get('/cookies', (req, res) => res.render('cookies'));
 app.get('/terms-and-conditions', (req, res) => res.render('terms'));
 
+// 404's
+app.use(require('hof').middleware.notFound({
+  logger: require('./lib/logger'),
+  translate: i18n.translate.bind(i18n),
```

Comment:
worth noting - its a good idea to include all `require()` calls at the top of the file. This is firstly because it is easy to see dependencies without trawling through the file, but more importantly, when `import module from 'module'` is introduced, any imports in the file are automatically hoisted to the top of the file so things like this will break